site/_data/templates.yaml

```diff
@@ -315,6 +315,9 @@ If you need an example of this behavior, check out the login page at https://www
   path: /oauth2/device
   version: 1.11.0
   variables:
+    - name: devicePendingIdPLink
+      type: PendingIdPLink
+      description: The PendingIdpLink that is generated when using the device OAuth flow.
```

Comment:
We should probably add more description here to let the user know when it will be available and why.